File: Latihan B/PYTHON/Disk.py
class Disk:

    __type = ""
    __capacity = ""
    __price = 0

    # konstruktor
    def __init__(self, type = "", capacity = "", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # set untuk semua variable
    def setType(self, type):
        self.__type = type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    # get untuk semua variable
    def getType(self):
        return self.__type

    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
    
    # tampilan untuk disk
    def printDisk(self):
        print("  Tipe Disk          : " + str(self.getType()))
        print("  Kapasitas Disk     : " + str(self.getCapacity()))
        print("  Harga Disk         : " + str(self.getPrice()))

File: Latihan A/C++/Disk.cpp

class Disk{
    private:
        int price, capacity;
        string type;
    public:

        Disk(){

        }
        Disk(int price, int capacity, string type){
            this->price = price;
            this->type = type;
            this->capacity = capacity;
        }

        //set semua variaber
        void setPriced(int price){
            this->price = price;
        }

        void setTyped(string type){
            this->type = type;
        }

        void setCapacityd(int capacity){
            this->capacity = capacity;
        }

        //get semua variabel    
        int getPriced(){
            return this->price;
        }

        string getTyped(){
            return this->type;
        }

        int getCapacityd(){
            return this->capacity;
        }

        //tampilan untuk disk
        void printDisk(){
            cout <<"  Tipe Disk          : " << this->type << endl;
            cout <<"  Kapasitas Disk     : " << this->capacity << endl;
            cout <<"  Harga Disk         : " << this->price << endl;
        }
        ~Disk(){
            
        }
};

